vignettes/CARNIVAL-vignette.Rnw

```diff
@@ -424,47 +430,100 @@ The solution network of this small real-case application is as shown in
 \label{fig:real_case}
 \end{figure*}
 
+\subsection{Toy Example - 3}
```

Comment:
tim-point toy example

Reply:
fixed in https://github.com/saezlab/CARNIVAL-Bioconductor-Dev/pull/10/commits/4843ec4ada3bafb462889c6b7e4d37830db67976